blog/20121227thank-you-first-colo.md

```diff
@@ -10,7 +10,7 @@
 
 ![FirstColo](/assets/img/blog/first-color.gif)
 
-[FirstColo](http://www.first-colo.net/EN/), operate a modern data center in Germany, and
+[FirstColo](https://www.first-colo.net/EN/), operate a modern data center in Germany, and
```

Comment:
https://www.first-colo.net/EN/ shows error 404

I think you can use https://www.first-colo.net/en/ it works

Reply:
Done